docs/_sidebar.md

```diff
@@ -1,3 +1,5 @@
+<a href="./playground"><button style="color:#FFFFFF;font-size:1rem;background-color:#2196f3;border: 2px #999 solid;border-radius: 5px;padding: 5px 10px;width: 80%; margin: 5px 10%; cursor:pointer;">Query Playground</button></a>
```

Comment:
I see you want a button, but if you just wanted to provide an external link that's possible like this: https://github.com/dhis2/ui-core/commit/561a0a6f38393cdae41bf6d73f9a7faf790241a8#diff-aee3e3a05f8f0a70aa7a8132bee2fddcR3

```
- [**Query Playground**](/playground/ ':ignore Title for the link can be put here')
```

Reply:
Yeah that's fair - I did want a button, but could be convinced it's superfluous.  I could also use a button in a markdown link with `:ignore`, to get both